hyperspy/_signals/lazy.py

```diff
@@ -268,6 +347,11 @@ def _get_navigation_chunk_size(self):
         nav_chunks = tuple([self.data.chunks[i] for i in sorted(nav_axes)])
         return nav_chunks
 
+    def _get_signal_chunk_size(self):
+        sig_axes = self.axes_manager.signal_indices_in_array
+        sig_chunks = tuple([self.data.chunks[i] for i in sorted(sig_axes)])
```

Comment:
is the sorted, necessary here? `axes_manager.signal_indices_in_array` could be correct?